anvil/src/udev.rs

```diff
@@ -244,14 +244,7 @@ pub fn run_udev() {
 
     let gpus = GpuManager::new(GbmGlesBackend::with_factory(|egl_display| {
         let egl_context = EGLContext::new_with_priority(egl_display, ContextPriority::High)?;
-        let disable_color_transform = std::env::var("ANVIL_DISABLE_COLOR_TRANSFORM").is_ok();
-        unsafe {
-            let capabilities = GlesRenderer::supported_capabilities(&egl_context)?
-                .into_iter()
-                .filter(|c| !disable_color_transform || *c != Capability::ColorTransformations);
-
-            Ok(GlesRenderer::with_capabilities(egl_context, capabilities)?)
-        }
+        unsafe { Ok(GlesRenderer::new(egl_context)?) }
```

Comment:
This could now use `GbmGlesBackend::with_context_priority(ContextPriority::High)`

Reply:
Good call, fixed.